pkgs/tools/misc/redtimer/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, makeWrapper
+, ruby
+, bundlerEnv
+, livesplit-core
+, ncurses
+, toilet
+}:
+
+let
+  gems = bundlerEnv {
+    name = "redtimer-env";
+    gemdir = ./.;
+    inherit ruby;
+  };
+in
+stdenv.mkDerivation rec {
+  pname = "redtimer";
+  version = "unstable-2021-08-01";
```

Comment:
Please either use a stable version or add a comment that explains why we package the unstable one.